refinery/core/models.py

```diff
@@ -2178,14 +2147,12 @@ def _nodecollection_delete(sender, instance, **kwargs):
     """Finds all subclasses related to a DataSet's NodeCollections and deletes
     all FileStoreItem instances associated with the DataSet
     """
-    nodes = Node.objects.filter(study=instance)
-    for node in nodes:
-        try:
-            FileStoreItem.objects.get(uuid=node.file_uuid).delete()
-        except (FileStoreItem.DoesNotExist,
-                FileStoreItem.MultipleObjectsReturned) as exc:
-            logger.debug("Could not delete FileStoreItem with UUID '%s': %s",
-                         node.uuid, exc)
+    if type(instance) == Study:
```

Comment:
Investigations are NodeCollections as well, but they get removed when a Study does due to the ForeignKey rel.